### ISSUE TITLE
Microsoft Edge doesn't start using Selenide 5.6.0

### ISSUE THREAD

**Issue by kgyorgy82** [selenide/selenide#1039]
## The problem
I just upgraded from Selenide 5.3.1 to 5.6.0. Everything is fine except running tests in Microsoft Edge. The browser doesn't start, selenide throws the exception below.

## Details
[main] INFO com.codeborne.selenide.impl.WebDriverThreadLocalContainer - No webdriver is bound to current thread: 1 - let's create a new webdriver
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Using msedgedriver pre-installed (since Microsoft Edge 44 is installed in your machine)
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Exporting webdriver.edge.driver as C:\WINDOWS\System32\MicrosoftWebDriver.exe
[14:17:53.446] - Listening on http://127.0.0.1:31912/
[14:17:53.446] - Listening on http://[::1]:31912/
[14:17:54.280] - Invalid capabilities

org.openqa.selenium.InvalidArgumentException: The specified arguments passed to the command are invalid.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-JLIA7J5', ip: '10.10.83.151', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.5'
Driver info: driver.version: SelenideDriver
selenide.url: http://xyz.hu
selenide.baseUrl: http://xyz.hu

remote stacktrace: 

	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:141)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:130)
	at com.codeborne.selenide.webdriver.EdgeDriverFactory.createEdgeDriver(EdgeDriverFactory.java:34)
	at com.codeborne.selenide.webdriver.EdgeDriverFactory.create(EdgeDriverFactory.java:18)
	at com.codeborne.selenide.webdriver.WebDriverFactory.lambda$createWebDriver$1(WebDriverFactory.java:51)
	at java.base/java.util.Optional.map(Optional.java:265)
	at com.codeborne.selenide.webdriver.WebDriverFactory.createWebDriver(WebDriverFactory.java:51)
	at com.codeborne.selenide.drivercommands.CreateDriverCommand.createDriver(CreateDriverCommand.java:39)
	at com.codeborne.selenide.impl.WebDriverThreadLocalContainer.createDriver(WebDriverThreadLocalContainer.java:117)
	at com.codeborne.selenide.impl.WebDriverThreadLocalContainer.getAndCheckWebDriver(WebDriverThreadLocalContainer.java:111)
	at com.codeborne.selenide.WebDriverRunner.getAndCheckWebDriver(WebDriverRunner.java:89)
	at com.codeborne.selenide.impl.StaticDriver.getAndCheckWebDriver(StaticDriver.java:47)
	at com.codeborne.selenide.SelenideDriver.getAndCheckWebDriver(SelenideDriver.java:150)
	at com.codeborne.selenide.drivercommands.Navigator.navigateTo(Navigator.java:65)
	at com.codeborne.selenide.drivercommands.Navigator.open(Navigator.java:28)
	at com.codeborne.selenide.SelenideDriver.open(SelenideDriver.java:66)
	at com.codeborne.selenide.Selenide.open(Selenide.java:40)
	at hu.icellmobilsoft.sqa.afw.DriverFactory.initDriver(DriverFactory.java:95)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.navigateToSite(CustomerBase.java:40)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.getStartScreen(CustomerBase.java:53)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.getStartScreen(CustomerBase.java:64)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.models.User.loginToHomeScreen(User.java:652)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.models.User.goToCustomerScreen(User.java:70)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.tests.InvoiceFilterOutboundTests.verifyOutboundInvoiceVatAmountManyInvoices(InvoiceFilterOutboundTests.java:248)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)

## Tell us about your environment
* **Selenide Version**: Selenide 5.6.0
* **Chrome\Firefox\IE Version**: 
"Microsoft Edge 44.18362.449.0
Microsoft EdgeHTML 18.18362"
or
"Microsoft Edge 44.18362.449.0
Microsoft EdgeHTML 18.18363"

* **Browser Driver Version**: version 44
* **Selenium Version**: Selenide 5.6.0
* **OS Version**: Win 10

## Code To Reproduce Issue [ Good To Have ]
Please remember that with sample code it's easier to reproduce the bug.

**Comment by SeleniumTestAB** [comment]
[14:17:54.280] - Invalid capabilities

org.openqa.selenium.InvalidArgumentException: The specified arguments passed to the command are invalid.

Could you send capabilities used in tests? Also do you use WebDriverRunner.setWebDriver()? or WebDriverProvider Interface?

**Comment by kgyorgy82** [comment]
I use the following method to initialize driver:

 Configuration.remote = cfgHelper.getIsRemote() ? cfgHelper.getRemoteServer().toString() : null;
 Configuration.browser = cfgHelper.getBrowserType();
Configuration.startMaximized = true;
Configuration.driverManagerEnabled = cfgHelper.getIsLocal();
Configuration.headless = cfgHelper.getIsHeadless();
Configuration.timeout = appConf.timeout().longValue();
Configuration.baseUrl = url.toString();
clearBrowserCache();
open(url.toString());

I have no custom capabilities set.

**Comment by SeleniumTestAB** [comment]
@kgyorgy82 for me it looks like something is wrong with Edge Driver itself than Selenide itself. Could you once more downgrade Selenide to lets say 5.5.1, and see what will happend?

**Comment by kgyorgy82** [comment]
@SeleniumTestAB I tried the version recommended by you (v 5.5.1) as well as the latest version (v.5.6.1) and got almost the same result.

[main] INFO com.codeborne.selenide.impl.WebDriverThreadLocalContainer - No webdriver is bound to current thread: 1 - let's create a new webdriver
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Using msedgedriver 80.0.361.33 (since Microsoft Edge 80 is installed in your machine)
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Exporting webdriver.edge.driver as C:\Users\user\.m2\repository\webdriver\edgedriver\win64\x64\80.0.361.33\msedgedriver.exe
Starting MSEdgeDriver 80.0.361.33 (22b78b8e452f4bd53829bff6b881ed90a74debcb) on port 47146
Only local connections are allowed.
Please protect ports used by MSEdgeDriver and related test frameworks to prevent access by malicious code.

org.openqa.selenium.WebDriverException: unknown error: cannot find MSEdge binary
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-JLIA7J5', ip: '10.130.7.37', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.5'
Driver info: driver.version: SelenideDriver
selenide.url: http://hubphq-icon-vapp-d001.icellmobilsoft.hu
selenide.baseUrl: http://hubphq-icon-vapp-d001.icellmobilsoft.hu
remote stacktrace: Backtrace:
	Ordinal0 [0x00007FF7535AB892+2144402]
	Ordinal0 [0x00007FF7534E6552+1336658]
	Ordinal0 [0x00007FF75343F295+651925]
	Ordinal0 [0x00007FF7533B0EF6+69366]
	Ordinal0 [0x00007FF7533AEE52+61010]
	Ordinal0 [0x00007FF7533D805D+229469]
	Ordinal0 [0x00007FF7533D52CF+217807]
	Ordinal0 [0x00007FF7533B704F+94287]
	Ordinal0 [0x00007FF7533B820E+98830]
	Ordinal0 [0x00007FF7534FF5B1+1439153]
	GetHandleVerifier [0x00007FF753670059+666233]
	GetHandleVerifier [0x00007FF75366FDE1+665601]
	GetHandleVerifier [0x00007FF75368674C+758124]
	GetHandleVerifier [0x00007FF753670868+668296]
	Ordinal0 [0x00007FF7534F628E+1401486]
	Ordinal0 [0x00007FF753501196+1446294]
	Ordinal0 [0x00007FF75351D17D+1560957]
	BaseThreadInitThunk [0x00007FFB5AAF7BD4+20]
	RtlUserThreadStart [0x00007FFB5B4ECED1+33]


	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:141)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:130)
	at com.codeborne.selenide.webdriver.EdgeDriverFactory.createEdgeDriver(EdgeDriverFactory.java:34)
	at com.codeborne.selenide.webdriver.EdgeDriverFactory.create(EdgeDriverFactory.java:18)
	at com.codeborne.selenide.webdriver.WebDriverFactory.lambda$createWebDriver$1(WebDriverFactory.java:51)
	at java.base/java.util.Optional.map(Optional.java:265)
	at com.codeborne.selenide.webdriver.WebDriverFactory.createWebDriver(WebDriverFactory.java:51)
	at com.codeborne.selenide.drivercommands.CreateDriverCommand.createDriver(CreateDriverCommand.java:39)
	at com.codeborne.selenide.impl.WebDriverThreadLocalContainer.createDriver(WebDriverThreadLocalContainer.java:126)
	at com.codeborne.selenide.impl.WebDriverThreadLocalContainer.getAndCheckWebDriver(WebDriverThreadLocalContainer.java:120)
	at com.codeborne.selenide.WebDriverRunner.getAndCheckWebDriver(WebDriverRunner.java:89)
	at com.codeborne.selenide.impl.StaticDriver.getAndCheckWebDriver(StaticDriver.java:47)
	at com.codeborne.selenide.SelenideDriver.getAndCheckWebDriver(SelenideDriver.java:150)
	at com.codeborne.selenide.drivercommands.Navigator.navigateTo(Navigator.java:65)
	at com.codeborne.selenide.drivercommands.Navigator.open(Navigator.java:28)
	at com.codeborne.selenide.SelenideDriver.open(SelenideDriver.java:66)
	at com.codeborne.selenide.Selenide.open(Selenide.java:41)
	at hu.icellmobilsoft.sqa.afw.DriverFactory.initDriver(DriverFactory.java:95)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.navigateToSite(CustomerBase.java:40)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.getStartScreen(CustomerBase.java:53)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.getStartScreen(CustomerBase.java:64)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.models.User.switchToLoginScreen(User.java:601)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.tests.LoginTests.verifyFilledLoginScreenValues(LoginTests.java:21)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)

**Comment by kgyorgy82** [comment]
@SeleniumTestAB I tried running Edge with the recommended version (v5.5.1) as  well as the latest one. (v5.6.1) Got this result, which is not exactly the same as before was.

[main] INFO com.codeborne.selenide.impl.WebDriverThreadLocalContainer - No webdriver is bound to current thread: 1 - let's create a new webdriver
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Using msedgedriver 80.0.361.33 (since Microsoft Edge 80 is installed in your machine)
[main] INFO io.github.bonigarcia.wdm.WebDriverManager - Exporting webdriver.edge.driver as C:\Users\user\.m2\repository\webdriver\edgedriver\win64\x64\80.0.361.33\msedgedriver.exe
Starting MSEdgeDriver 80.0.361.33 (22b78b8e452f4bd53829bff6b881ed90a74debcb) on port 24221
Only local connections are allowed.
Please protect ports used by MSEdgeDriver and related test frameworks to prevent access by malicious code.

org.openqa.selenium.WebDriverException: unknown error: cannot find MSEdge binary
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-JLIA7J5', ip: '10.130.7.37', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.5'
Driver info: driver.version: SelenideDriver
selenide.url: http://xxx.hu
selenide.baseUrl: http://xxx.hu
remote stacktrace: Backtrace:
	Ordinal0 [0x00007FF7535AB892+2144402]
	Ordinal0 [0x00007FF7534E6552+1336658]
	Ordinal0 [0x00007FF75343F295+651925]
	Ordinal0 [0x00007FF7533B0EF6+69366]
	Ordinal0 [0x00007FF7533AEE52+61010]
	Ordinal0 [0x00007FF7533D805D+229469]
	Ordinal0 [0x00007FF7533D52CF+217807]
	Ordinal0 [0x00007FF7533B704F+94287]
	Ordinal0 [0x00007FF7533B820E+98830]
	Ordinal0 [0x00007FF7534FF5B1+1439153]
	GetHandleVerifier [0x00007FF753670059+666233]
	GetHandleVerifier [0x00007FF75366FDE1+665601]
	GetHandleVerifier [0x00007FF75368674C+758124]
	GetHandleVerifier [0x00007FF753670868+668296]
	Ordinal0 [0x00007FF7534F628E+1401486]
	Ordinal0 [0x00007FF753501196+1446294]
	Ordinal0 [0x00007FF75351D17D+1560957]
	BaseThreadInitThunk [0x00007FFB5AAF7BD4+20]
	RtlUserThreadStart [0x00007FFB5B4ECED1+33]


	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
	at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
	at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.Spliterators$ArraySpliterator.tryAdvance(Spliterators.java:958)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:141)
	at org.openqa.selenium.edge.EdgeDriver.<init>(EdgeDriver.java:130)
	at com.codeborne.selenide.webdriver.EdgeDriverFactory.createEdgeDriver(EdgeDriverFactory.java:34)
	at com.codeborne.selenide.webdriver.EdgeDriverFactory.create(EdgeDriverFactory.java:18)
	at com.codeborne.selenide.webdriver.WebDriverFactory.lambda$createWebDriver$1(WebDriverFactory.java:51)
	at java.base/java.util.Optional.map(Optional.java:265)
	at com.codeborne.selenide.webdriver.WebDriverFactory.createWebDriver(WebDriverFactory.java:51)
	at com.codeborne.selenide.drivercommands.CreateDriverCommand.createDriver(CreateDriverCommand.java:39)
	at com.codeborne.selenide.impl.WebDriverThreadLocalContainer.createDriver(WebDriverThreadLocalContainer.java:117)
	at com.codeborne.selenide.impl.WebDriverThreadLocalContainer.getAndCheckWebDriver(WebDriverThreadLocalContainer.java:111)
	at com.codeborne.selenide.WebDriverRunner.getAndCheckWebDriver(WebDriverRunner.java:89)
	at com.codeborne.selenide.impl.StaticDriver.getAndCheckWebDriver(StaticDriver.java:47)
	at com.codeborne.selenide.SelenideDriver.getAndCheckWebDriver(SelenideDriver.java:150)
	at com.codeborne.selenide.drivercommands.Navigator.navigateTo(Navigator.java:65)
	at com.codeborne.selenide.drivercommands.Navigator.open(Navigator.java:28)
	at com.codeborne.selenide.SelenideDriver.open(SelenideDriver.java:66)
	at com.codeborne.selenide.Selenide.open(Selenide.java:40)
	at hu.icellmobilsoft.sqa.afw.DriverFactory.initDriver(DriverFactory.java:95)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.navigateToSite(CustomerBase.java:40)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.getStartScreen(CustomerBase.java:53)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.pageobject.CustomerBase.getStartScreen(CustomerBase.java:64)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.models.User.switchToLoginScreen(User.java:601)
	at hu.icellmobilsoft.sqa.icon.frontend.customer.tests.LoginTests.verifyFilledLoginScreenValues(LoginTests.java:21)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:648)
	at org.testng.TestRunner.run(TestRunner.java:505)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
	at org.testng.TestNG.runSuites(TestNG.java:1049)
	at org.testng.TestNG.run(TestNG.java:1017)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)

**Comment by miltonsiqueira** [comment]
I had something like that using `ie`browser . In `v5.5.1` works fine.

I'm guessing that this happened because of a `ACCEPT_INSECURE_CERTS` capability was added in `AbstractDriverFactory.java`.
https://github.com/selenide/selenide/commit/7bb9697766da7c8f2d58eb68269ed267b66fb492#diff-715bf2ca0e631406aa05c47f40a04e86R65-R70

"The IE driver does not allow you to accept self-signed certificates"  
https://github.com/SeleniumHQ/selenium/issues/4704#issuecomment-329539218

**Comment by miltonsiqueira** [comment]
This workaround works here.

```java
Configuration.browserCapabilities.setAcceptInsecureCerts(false);
```

**Comment by asolntsev** [comment]
Fixed in https://github.com/selenide/selenide/pull/1075

**Comment by mjevtovic** [comment]
Does this solve issues with Edge? I am having problems with running Edge with Selenide 5.8.0 and 5.7.0 (didn't try previous version)

This is error I am getting

```
java.lang.IllegalArgumentException: org.openqa.selenium.InvalidArgumentException: The specified arguments passed to the command are invalid.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'lt2016001.local', ip: '192.168.0.48', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.3', java.version: '1.8.0_25'
Driver info: driver.version: SelenideDriver
remote stacktrace: 

	at com.codeborne.selenide.webdriver.AbstractDriverFactory.createInstanceOf(AbstractDriverFactory.java:52)
	at com.codeborne.selenide.webdriver.DefaultDriverFactory.create(DefaultDriverFactory.java:16)

...
```
And my capabilities are

```
        capabilities.setBrowserName("microsoft edge");
        capabilities.setVersion("18");
```
Also tried with

```
        capabilities.setBrowserName("microsoft edge");
        capabilities.setVersion("18");
        capabilities.setAcceptInsecureCerts(false);
```

**Comment by asolntsev** [comment]
@mjevtovic Thank you for paying our attention to this. We had the same problem for Edge, not only for IE. 
Now it's fixed by https://github.com/selenide/selenide/pull/1084.
Will be released as Selenide 5.9.0 tonight.

**Comment by asolntsev** [comment]
@mjevtovic @BorisOsipov I discovered that the latest (chromium-based) Edge driver (e.g. version 80+) supports "ACCEPT_INSECURE_CERTS" property. It would be great to restore this setting for latest browser versions.
